core/state_transition.go

```diff
@@ -27,6 +27,10 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
+var (
+	Big0 = big.NewInt(0)
```

Comment:
Actually I think this is fine (although unexported). I've mentioned before that we should get rid of the global, common, "constants" especially in consensus critical code. If someone, accidentally, changes the int (set, add, mul, etc) it could lead to some serious issues.